src/fight.cpp

```diff
@@ -5698,8 +5711,10 @@ void perform_violence(void)
     }
 
     // You get no action if you're out of init.
-    if (GET_INIT_ROLL(ch) <= 0 && !IS_JACKED_IN(ch) && PRF_FLAGGED(ch, PRF_SEE_TIPS)) {
```

Comment:
Hah, this didn't just affect mobs, it was an exploit where someone could turn off tips and get unlimited initiative. Nice find.